<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>989</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>719</x>
      <y>49</y>
      <width>271</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>换乘路线：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>730</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>请选择：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>780</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>起始站：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>830</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>终点站：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>890</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>换乘策略：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>780</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>线路：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Sline">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>780</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>780</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>站点：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Sstation">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>780</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>830</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>线路：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Eline">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>830</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>830</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>站点：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Estation">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>830</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Transfer">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>890</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>最短时间优先</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>最少换乘优先</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_Transfer">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>780</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>换乘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DisplayOne">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>880</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>显示单独</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DisplayAll">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>880</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>显示全貌</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_DisplayOne">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>890</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>请选择</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>添加(A)</string>
    </property>
    <addaction name="actionAddNewLine"/>
    <addaction name="separator"/>
    <addaction name="actionAddNewStation"/>
    <addaction name="separator"/>
    <addaction name="actionAddNewEdge"/>
    <addaction name="separator"/>
    <addaction name="actionAddNew"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>工具(T)</string>
    </property>
    <addaction name="actionZoomUp"/>
    <addaction name="separator"/>
    <addaction name="actionZoomDown"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionAddNewLine">
   <property name="text">
    <string>添加新线路</string>
   </property>
  </action>
  <action name="actionAddNewStation">
   <property name="text">
    <string>添加新站点</string>
   </property>
  </action>
  <action name="actionAddNewEdge">
   <property name="text">
    <string>添加新连接</string>
   </property>
  </action>
  <action name="actionAddNew">
   <property name="text">
    <string>添加...</string>
   </property>
  </action>
  <action name="actionZoomUp">
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actionZoomDown">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
